src/Deserializers/LegacySnakDeserializer.php

```diff
@@ -41,11 +42,7 @@ public function deserialize( $serialization ) {
 				return new PropertySomeValueSnak( $serialization[1] );
 			case 'value':
 				return $this->deserializeValueSnak( $serialization );
-				// @codeCoverageIgnoreStart
-			default:
-				throw new LogicException();
```

Comment:
This is not just "dead code". It's there for a purpose. PHPStorm and other tools complain about this for a reason. The fact that you, when following all execution paths, are able to understand that no code path in the current implementation can reach this line does not mean all other people reading the code will understand this. More important: It does not mean these code paths will never change and never reach this line.

So still -1.